src/routes/ConsortiumManager/settings/users/general/PermissionSets/PermissionSetsCreate/PermissionSetsCreate.js

```diff
@@ -0,0 +1,17 @@
+import { useMemo } from 'react';
+import { useLocation } from 'react-router-dom';
+
+import { ConsortiumPermissionsSetForm } from '../ConsortiumPermissionsSetForm';
+import { useTenantPermissionSetMutations } from '../hooks';
+import { TENANT_ID_SEARCH_PARAMS } from '../constants';
+
+export const PermissionSetsCreate = () => {
+  const location = useLocation();
+  const tenantId = useMemo(() => new URLSearchParams(location.search).get(TENANT_ID_SEARCH_PARAMS), [location.search]);
+
+  const { createPermissionSet } = useTenantPermissionSetMutations(tenantId);
+
+  const onSave = (values) => createPermissionSet(values);
+
+  return <ConsortiumPermissionsSetForm onSave={onSave} />;
```

Comment:
I believe `onSave={createPermissionSet}` should work as well, in addition, it will prevent unnecessary re-renders since the reference to the mutation is passed instead of callback recreation on each render

Reply:
hm, it was what I asked above, seems not fixed yet